account_bank_statement_import_txt_xlsx/migrations/12.0.3.0.0/post-migration.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+UPDATE account_bank_statement_import_sheet_mapping
+    set header_lines_number = 1,
```

Comment:
Not a change request, yet it's better to keep same case of keywords everywhere (e.g. if it's `UPDATE` then it's `SET`, `NULL`, etc.)

Reply:
Thanks @alexey-pelykh for your code review.
OK modified